docs/Projects/TAD/ADRs/0004-software-stack.md

```diff
@@ -45,15 +46,17 @@ The Python backend will use the following key dependencies:
 
 * [Granian](https://github.com/emmett-framework/granian) as HTTP server
 * [Pydantic](https://docs.pydantic.dev/latest/) for data validation
-* TODO: decide on [FastAPI](https://fastapi.tiangolo.com/) or
-  [Flask](https://flask.palletsprojects.com/en/3.0.x/)
+* [FastAPI](https://fastapi.tiangolo.com/) for REST/API/HTML communication
+* [JinjaX](https://jinjax.scaletti.dev/) for templating
 
 ### Frontend
 
 We will use serverside rendering of HTML, based on [HTMX](https://htmx.org/) and
 TODO: decide what template framework to use, like [Jinja](https://jinja.palletsprojects.com/en/3.1.x/).
 This combination is expected to work intuitive and fast.
 
+TODO: decide how to implement / handle multi-lingual support
```

Comment:
great question!

Jinja2 has this extension: https://jinja.palletsprojects.com/en/3.1.x/extensions/#i18n-extension

A quick search didn't yield this for JinjaX (yet)